assets/scripts/git.js

```diff
@@ -1,20 +0,0 @@
-(() => {
```

Comment:
Dit kan niet weg. Het menu wordt dynamisch opgebouwd op basis van branches.json in de root van het project. Daar staat altijd de meest actuele verzameling van branches. Omdat het branches menu dynamisch wordt opgebouwd vanuit dit bestand is het branches menu hetzelfde over alle features.

zie ook regel  109 (script(src=relative.basePath + 'branches.js' charset='UTF-8'))

Reply:
Ik heb git.js teruggezet en aangepast.